packages/devtools_testing/pubspec.yaml

```diff
@@ -2,16 +2,16 @@ name: devtools_testing
 description: Package of shared testing code for Dart DevTools.
 # Note: this version should only be updated by running tools/update_version.sh
 # that updates all versions of packages from packages/devtools.
-version: 0.2.3
+version: 0.2.4
 
 homepage: https://github.com/flutter/devtools
 
 environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  devtools_app: 0.2.3
-  devtools_shared: 0.2.3
+  devtools_app: 0.2.4
```

Comment:
this will also get updated when we publish. note all the versions are locked to dev dependencies when we run and test.